Robert Khromov/src/app/components/Posts.js

```diff
@@ -0,0 +1,54 @@
+import React, {Component} from 'react';
+import Post from "../components/Post";
+import {addPost, getPosts} from '../actions/post';
+import {connect} from 'react-redux';
+
+class Posts extends Component {
+
+    constructor(props) {
+        super(props);
+        this.createNewPost = this.createNewPost.bind(this);
+    }
+
+    componentDidMount() {
+        getPosts();
+    }
+
+    createNewPost() {
+        addPost({
+            id: Math.round(Math.random() * 1000),
+            title: 'ЪУЪ',
+            body: 'ЪОЪ'
+        })
+    }
+
+    render() {
+
+        if (this.props.fetching) {
+            return <h1>Загрузка ...</h1>
+        }
+
+        let posts = this.props.posts.map((post) => {
+            return <Post key={post.id} {...post}/>
+        });
+
+        return (
+            <div>
+                <h1>
+                    Посты
+                    <button type="button" className="btn btn-primary" style={{marginLeft: "30px"}}
+                            onClick={this.createNewPost}>Добавить</button>
+                </h1>
+                {posts}
+            </div>
+        )
+    };
+}
+
+function mapStoreToProps(state) {
```

Comment:
```suggestion
function mapStateToProps(state) {
```
понимаю, что вы хотите повысить читабельность ( и, действительно, как-то логичнее указывать `Store`, а не `State`), но вы привыкните к данной записи и вам будет легче распознать в чужом коде подключение глобального `State'a` к компоненту.